### PR TITLE
m2 version history

### PR DESCRIPTION
**Pull request by hflynn**
See https://trello.com/c/KK3DJFbE/20-version-history-prs

I've maybe summarised too much and missed details that are useful for devs, happy to expand if so.

Staged at http://www.openmicroscopy.org/site/support/bio-formats5.3-staging/about/whats-new.html

### CODE REVIEW

File: docs/sphinx/about/whats-new.txt [hunk 1]
```diff
@@ -1,6 +1,25 @@
 Version history
 ===============
 
+5.3.0-m2 (2016 November 23)
```
Comment (sbesson) [inline docs/sphinx/about/whats-new.txt]: One question here: are we listing each milestone separately or integrating them together? I would favor the second option as developers will typically upgrade to 5.3.0 and want to list all changes.

File: docs/sphinx/about/whats-new.txt [hunk 2]
```diff
@@ -1,6 +1,25 @@
...
+---------------------------
+
+Fixes and clean up:
+
+* clean up of bundles, packages and build targets to reflect decoupling work
+* POM repositories clean up
+* updated developer documentation on building Bio-Formats
+* updated testing packages
+* updated build versioning from Maven
```
Comment (sbesson) [inline docs/sphinx/about/whats-new.txt]: I think we could expand on this to explain that 1- we unified the versioning strategy, 2- we reviewed the meta information stored in the manifest of each JAR and 3- we are now making introspecting this meta-information in the `FormatTools` API to retrieve the version and revision number. Does that correctly represent your contribution @rleigh-codelibre ?

File: docs/sphinx/about/whats-new.txt [hunk 3]
```diff
@@ -1,6 +1,25 @@
...
+
+Further decoupling:
+
+* removed stubs and metakit and replaced them with jars from Maven Central
```
Comment (sbesson) [inline docs/sphinx/about/whats-new.txt]: As we did previously, we might want to add extra information about the repository and `groupId/artifactId`

File: docs/sphinx/about/whats-new.txt [hunk 4]
```diff
@@ -1,6 +1,25 @@
...
+
+File format support:
+
+* added support for JPEG-XR compressed CZI data
```
Comment (sbesson) [inline docs/sphinx/about/whats-new.txt]: I would minimally mention `ome:jxrlib` as a new dependency of Bio-Formats. Is it worth linking to the blog post/press announcement? @melissalinkert @chris-allan : anything in particular to mention on this page or should we generically review as part of the documentation of the CZI work?

File: docs/sphinx/about/whats-new.txt (hunk 1 above)
Comment (hflynn) [inline docs/sphinx/about/whats-new.txt]: I assumed we'd amalgamate the list for the 5.3.0 release but I can do it as we go along if you prefer.

Comment (sbesson) [inline docs/sphinx/about/whats-new.txt]: Either way. The advantage of combining early is that we could cut some redundancy e.g. I wonder whether the "clean up..."  should be mentioned at all in a combined history.

**Comment by hflynn** [comment]
Updated merging both milestones histories into one entry, keeping categories from m1 and adding in additional detail including the things noted by @sbesson and a link to the Glencoe press announcement. Hopefully better now!

**Comment by sbesson** [comment]
Staged at https://www.openmicroscopy.org/site/support/bio-formats5.3-staging/about/whats-new.html and looks good to me. A couple of nitpicks unrelated to this PR that can be addressed post 5.3.0-m2:
- there is no `org.openmicroscopy:xsd-fu` artifact only `specification` and `ome-xml`
- should we include `ome:jxrlib` in the table of dependencies or rather in the components page?
- the quote format is inconsistent between the `org.openmicroscopy:*` components and `ome:jxrlib`

**Comment by hflynn** [comment]
Quote format is related to this PR, I'll fix that now

**Comment by dgault** [comment]
Having done a quick proof read it appears to read and display fine to me

**Comment by hflynn** [comment]
Actually, do you want me to remove the xsd-fu artifact that doesn't exist now too @sbesson ?

**Comment by hflynn** [comment]
2nd point not addressed here is covered by https://trello.com/c/HeoKWl5H/52-add-documentation-for-reading-czi-files-with-jpeg-xr

**Pull request merged**